docs/interfaces.md

```diff
@@ -460,6 +460,13 @@ processes on the system.
 
 * Auto-Connect: no
 
+### system-power-control
```

Comment:
I'll point out that the desktop system indicator has a menu item called "Shut Down..." which offers the user the ability to shutdown or reboot.  So we could call this "shutdown" or "reboot".  I don't have a strong feeling either way, but just wanted to note that for the record.